OpenRA.Mods.Common/Widgets/WidgetUtils.cs

```diff
@@ -12,6 +12,7 @@
 using System;
 using System.Linq;
 using OpenRA.Graphics;
+using OpenRA.Mods.Common.Widgets;
 using OpenRA.Primitives;
 
 namespace OpenRA.Widgets
```

Comment:
This needs to change to `OpenRA.Mods.Common.Widgets` as this is no longer in `OpenRA.Game`. The `using` above can then also go away, and a few other classes will need to add `using OpenRA.Mods.Common.Widgets`.

Reply:
Fixed.